gage_outages/outage_maps.R

```diff
@@ -9,23 +9,66 @@ library(rgeos)
 library(ggplot2)
 library(readxl)
 library(sf)
+library(googlesheets)
 
 # sbtools::authenticate_sb()
 vizlab::authRemote('sciencebase')
-# Site from google sheet at 6:30am Thursday 10/25:
-# sites <- setDF(fread("site_list_thurs_morning.tsv", sep = "\t"))
-# 
-# sites$siteID_15 <- str_match( sites[[1]], "\\d{15}")[,1]
-# sites$siteID_10 <- str_match( sites[[1]], "\\d{10}")[,1]
-# sites$siteID_9 <- str_match( sites[[1]], "\\d{9}")[,1]
-# sites$siteID_8 <- str_match( sites[[1]], "\\d{8}")[,1]
+
+################################
+# Get Latest sites
+################################
+token <- gs_auth(cache = FALSE)
+title_2 <- gs_title("GOES/DA ISSUE STARTING 2018-10-20")
+current_site_list <- gs_read(title_2, range = "A5:Q1000")
+
+current_site_list$siteID_15 <- stringr::str_match( current_site_list[[1]], "\\d{15}")[,1]
+current_site_list$siteID_10 <- stringr::str_match( current_site_list[[1]], "\\d{10}")[,1]
+current_site_list$siteID_9 <- stringr::str_match( current_site_list[[1]], "\\d{9}")[,1]
+current_site_list$siteID_8 <- stringr::str_match( current_site_list[[1]], "\\d{8}")[,1]
```

Comment:
doesn't there need to be a non-digit at the end of these regexes?

Reply:
I took it out because a few sites *don't* have a space (sigh...). The logic here works though as long as we don't re-arrange any of the lines.